backend-worker/worker.py

```diff
@@ -576,7 +576,7 @@ def _to_hex(r, g, b, a):
     get_classnames_from_raster_attr_table(NLCD_RASTER_PATH).items()))
 
 
-def make_thumbnail(pattern_wkt_epsg3857, colors_dict, target_thumnail_path,
+def make_thumbnail(pattern_wkt_epsg3857, colors_dict, target_thumbnail_path,
```

Comment:
Oh my gosh, I'm so sorry I hadn't caught this typo!  I'm actually a bit surprised, I would have thought my spellcheck would have.